### ISSUE TITLE
Image size does not match in Clipart1k

### ISSUE THREAD

**Issue by XuYunqiu** [naoto0804/cross-domain-detection#39]
Hi, thanks for sharing the datasets. I find a mismatch issue about an image and its annotation. 

The image `JPEGImages/681633840.jpg` in Clipart1k dataset has a size of 300*213. But its annotation `Annotations/681633840.xml` shows it should have a width of 600 and a height of 594.

I try to resize the image to the size provided in the annotation. However, bounding boxes cannot cover objects correctly.

![image](https://user-images.githubusercontent.com/23272227/125621028-3a4aa0b1-a4e2-4197-9e8b-e22fd25f9f07.png)

**Comment by pengjw23** [comment]
I have the same question with you, had you solved this problem?

**Comment by XuYunqiu** [comment]
> I have the same question with you, had you solved this problem?

No, I didn't. I just resized the image according to its annotation, and used its image-level labels for training.
I find several images whose size are inconsistent with their annotations. I have listed them in https://github.com/XuYunqiu/H2FA_R-CNN/blob/main/mismatch_anno_list.txt
